pkg/terraform/schema/schema.json

```diff
@@ -56,6 +24,12 @@
     "terraform": {
       "type": "object",
       "properties": {
+        "arguments": {
```

Comment:
Does uninstall not share this same reference because it doesn't have the same set of fields (it looks like arguments is the big difference jumping out at me)? Or was that just an oversight?

Reply:
Indeed, uninstall has a different set of fields (no `input` flag)... though maybe we can consolidate in the schema but verify in code?  (That `input` is not valid for an uninstall action, for instance.)

Reply:
Nope, I was just asking because I didn't know, not to say we should do anything. 😀 

We shouldn't consolidate in schema ever because that messes up autocomplete and makes people think that something is valid when it isn't. So no worries, what you have is fine. I am just asking questions to learn is all. I'll make sure to make my questions more clear in the future. 👍